docs/source/tracking.rst

```diff
@@ -157,7 +157,7 @@ Tracking UI
 The Tracking UI lets you visualize, search and compare runs, as well as download run artifacts or
 metadata for analysis in other tools. If you have been logging runs to a local ``mlruns`` directory,
 simply run ``mlflow ui`` in the directory above it, and it will load the corresponding runs.
-Alternatively, the :ref:`MLflow server <tracking_server>` serves the same UI.
+Alternatively, the :ref:`mlflow server <tracking_server>` serves the same UI, and enables remote storage of run artifacts.
```

Comment:
I think MLflow was ok here because this is not formatted as code (despite the confusing tick marks, which look similar to the `mlflow ui` tick marks above)